src/main/java/ch/njol/skript/effects/EffSendTitle.java

```diff
@@ -0,0 +1,117 @@
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+
+@Name("Title - Send")
+@Description({"Sends a title/subtitle to the given player(s) with optional fadein/stay/fadeout times for Minecraft versions 1.11 and above.",
+		"Note: if no input is given for the title/subtitle or the times," +
+		"it will keep the ones from the last title sent, use the <a href='effects.html#EffResetTitle'>reset title</a> effect to restore the default values."})
+@Examples({"send title \"Competition Started\" with subtitle \"Have fun, Stay safe!\" to player for 5 seconds",
+		"send title \"Hi %player%\" to player", "send title \"Loot Drop\" with subtitle \"starts in 3 minutes\" to all players",
+		"send title \"Hello %player%!\" with subtitle \"Welcome to our server\" to player for 5 seconds with fadein 1 second and fade out 1 second",
+		"send subtitle \"Party!\" to all players"})
+@Since("INSERT VERSION")
+public class EffSendTitle extends Effect {
+	
+	private final static boolean TIME_SUPPORTED = Skript.methodExists(Player.class,"sendTitle", String.class, String.class, int.class, int.class, int.class);
+	
+	static {
+		if (TIME_SUPPORTED)
+			Skript.registerEffect(EffSendTitle.class,
+					"send title %string% [with subtitle %-string%] [to %players%] [for %-timespan%] [with fade[(-| )]in %-timespan%] [(and|with) fade[(-| )]out %-timespan%]",
+					"send subtitle %string% [to %players%] [for %-timespan%] [with fade[(-| )]in %-timespan%] [(and|with) fade[(-| )]out %-timespan%]");
+		else
+			Skript.registerEffect(EffSendTitle.class, "send title %string% [with subtitle %-string%] [to %players%]",
+												"send subtitle %string% [to %players%]");
+		
+	}
+	
+	@Nullable
+	private Expression<String> title;
+	@Nullable
+	private Expression<String> subtitle;
+	@SuppressWarnings("null")
+	private Expression<Player> recipients;
+	@Nullable
+	private Expression<Timespan> fadeIn, stay, fadeOut;
+	
+	@SuppressWarnings({"unchecked", "null"})
+	@Override
+	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
+		title = matchedPattern == 0 ? (Expression<String>) exprs[0] : null;
+		subtitle = matchedPattern == 0 ? (Expression<String>) exprs[1] : (Expression<String>) exprs[0];
+		recipients = matchedPattern == 0 ? (Expression<Player>) exprs[2] : (Expression<Player>) exprs[1];
+		if (TIME_SUPPORTED) {
+			stay = matchedPattern == 0 ? (Expression<Timespan>) exprs[3] : (Expression<Timespan>) exprs[2];
+			fadeIn = matchedPattern == 0 ? (Expression<Timespan>) exprs[4] : (Expression<Timespan>) exprs[3];
```

Comment:
This could be changed to:
```java
stay = (Expression<Timespan>) exprs[3 - matchedPattern];
```
Same with the others

Reply:
Fixed